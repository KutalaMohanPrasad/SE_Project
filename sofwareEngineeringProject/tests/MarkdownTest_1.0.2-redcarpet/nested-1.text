## Nested

*	Tab
	*	Tab
		*	Tab

Here's another:

1. First
2. Second:
	* Fee
	* Fie
	* Foe
3. Third
