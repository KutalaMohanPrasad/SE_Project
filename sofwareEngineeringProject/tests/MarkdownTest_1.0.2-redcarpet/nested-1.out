<h2>Nested</h2>

<ul>
<li>Tab
<ul>
<li>Tab
<ul>
<li>Tab</li>
</ul></li>
</ul></li>
</ul>

<p>Here's another:</p>

<ol>
<li>First</li>
<li>Second:
<ul>
<li>Fee</li>
<li>Fie</li>
<li>Foe</li>
</ul></li>
<li>Third</li>
</ol>